secrets.template

```diff
@@ -0,0 +1,44 @@
+## Discord Vars ##
+# These variables are used to setup the actual Discord bot.
+DISCORD_TOKEN=
+COMMAND_PREFIX=!
+UNKNOWN_COMMAND_EMOJI=!?
+##################
+
+## Database Vars ##
+# These are variables are used to define where your Database is stored and the login information.
+# These are variables are required.
+PG_HOST=
+PG_USER=
+PG_PWD=
+PG_DATABASE=
+###################
+
+## Music Vars ##
+# These are variables are used for the MusicCog to function.
+# Only needed if the MusicCog is enabled using the ENABLE_MUSIC variable.
+ENABLE_MUSIC=
+GOOGLE_API=
+################
+
+## Twitch Vars ##
+# These are variables are used for the TwitchCog to function.
+# Only needed if the TwitchCog is enabled using the ENABLE_TWITCH variable.
+ENABLE_TWITCH=
+TWITCH_CLIENT_ID=
+TWITCH_CLIENT_SECRET=
+TWITCH_SUB_SECRET=
+TWITCH_CALLBACK=
+#################
+
+## Twitter Vars ##
+# These variables are used for the TwitterCog to function.
+# Only needed if the TwitterCog is enabled using the ENABLE_TWITTTER variable.
+ENABLE_TWITTER=
+TWITTER_CONSUMER_KEY=
+TWITTER_CONSUMER_SECRET=
+TWITTER_ACCESS_TOKEN=
+TWITTER_ACCESS_TOKEN_SECRET=
+##################
+
+
```

Comment:
***Massive*** thanks for adding a template file with doc. It's good to get all our variable doc out of the readme.